sparse/tests/conftest.py

```diff
@@ -1,8 +1,9 @@
-import pytest
+import sparse
 
+import pytest
 
-@pytest.fixture(scope="session")
-def rng():
-    from sparse._utils import default_rng
 
-    return default_rng
```

Comment:
Probably shouldn't be deleted, I guess it's used in the tests?

Reply:
This is displayed that way only in GitHub. `sparse/tests/conftest.py` was moved to `sparse/pydata_backend/tests/conftest.py`, and then I added a new file, `sparse/tests/conftest.py`.

In the diff it shows it as if the original file was changed, but you can still find `def rng():` in `sparse/pydata_backend`.

Reply:
I see, in that case I'd merge the two files here... There should be only one `tests` directory at the top level.

Reply:
For transparency: We decided to have backend-specific tests into their own directories, moving things out as they encompassed multiple backends.